java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/transaction/PoTransaction.java

```diff
@@ -947,7 +969,8 @@ public final void processClosing(ChannelControl channelControl)
         }
 
         // Finally, close the session as requested
-        processAtomicClosing(poAtomicCommands, calypsoPo.getTransmissionMode(), channelControl);
+        processAtomicClosing(poAtomicCommands, poSecuritySettings.getRatificationMode(),
```

Comment:
poSecuritySettings.getRatificationMode() is optionnal

Reply:
Yes, processAtomicClosing is private.
processClosing has no ratification mode argument.
The default value for the ratification mode is defined in PoSecuritySettings.
We use the possibility to call processAtomicClosing with CLOSE_RATIFIED when doing multiple sessions.